### PR TITLE
core: stricter LR desktop metric thresholds

### LINKED ISSUE #6836: lr-desktop preset is too generous
ez 100's. 

We all think it could be a little more stern.

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Uses data from HTTPArchive desktop runs to get new score thresholds

## CNN Before
 image

## CNN After
image | 

**Related Issues/PRs**
fixes #6836

### CODE REVIEW

File: lighthouse-cli/cli-flags.js [hunk 1]
```diff
@@ -137,7 +137,6 @@ function getFlags(manualArgv) {
       // default values
       .default('chrome-flags', '')
       .default('output', ['html'])
-      .default('emulated-form-factor', 'mobile')
```
Comment (patrickhulce) [inline lighthouse-cli/cli-flags.js]: This was an erroneous double default that makes it impossible to run `lr-desktop-config` from the CLI

File: lighthouse-core/config/lr-desktop-config.js [hunk 2]
```diff
@@ -21,6 +21,19 @@ const config = {
...
+    {path: 'metrics/first-contentful-paint', options: {scorePODR: 800, scoreMedian: 1600}},
+    {path: 'metrics/first-meaningful-paint', options: {scorePODR: 800, scoreMedian: 1600}},
+    // 75th and 95th percentiles -> median and PODR
+    // SELECT QUANTILES(SpeedIndex, 21) FROM [httparchive:summary_pages.2018_12_15_desktop] LIMIT 1000
+    {path: 'metrics/speed-index', options: {scorePODR: 1100, scoreMedian: 2300}},
+    // 75th and 95th percentiles -> median and PODR
+    // SELECT QUANTILES(fullyLoaded, 21) FROM [httparchive:summary_pages.2018_12_15_desktop] LIMIT 1000
+    {path: 'metrics/interactive', options: {scorePODR: 2000, scoreMedian: 4500}},
+    {path: 'metrics/first-cpu-idle', options: {scorePODR: 2000, scoreMedian: 4500}},
+  ]
```
Comment (exterkamp) [inline lighthouse-core/config/lr-desktop-config.js]:
```suggestion
  ],
```

File: lighthouse-cli/cli-flags.js (hunk 1 above)
Comment (brendankenny) [reply lighthouse-cli/cli-flags.js]: was the cli flag winning over the config setting? Yeah, that seems bad...

Comment (brendankenny) [reply lighthouse-cli/cli-flags.js]: does `output` have the same problem? I don't see any other of these defaults in `SharedFlagsSettings`

File: lighthouse-core/config/lr-desktop-config.js [hunk 3]
```diff
@@ -21,6 +21,19 @@ const config = {
...
+    // SELECT QUANTILES(renderStart, 21) FROM [httparchive:summary_pages.2018_12_15_desktop] LIMIT 1000
+    {path: 'metrics/first-contentful-paint', options: {scorePODR: 800, scoreMedian: 1600}},
+    {path: 'metrics/first-meaningful-paint', options: {scorePODR: 800, scoreMedian: 1600}},
+    // 75th and 95th percentiles -> median and PODR
+    // SELECT QUANTILES(SpeedIndex, 21) FROM [httparchive:summary_pages.2018_12_15_desktop] LIMIT 1000
+    {path: 'metrics/speed-index', options: {scorePODR: 1100, scoreMedian: 2300}},
+    // 75th and 95th percentiles -> median and PODR
+    // SELECT QUANTILES(fullyLoaded, 21) FROM [httparchive:summary_pages.2018_12_15_desktop] LIMIT 1000
+    {path: 'metrics/interactive', options: {scorePODR: 2000, scoreMedian: 4500}},
+    {path: 'metrics/first-cpu-idle', options: {scorePODR: 2000, scoreMedian: 4500}},
```
Comment (brendankenny) [inline lighthouse-core/config/lr-desktop-config.js]: wooooo good luck desktop sites!

**Comment by exterkamp** [comment]
How do I downvote a review?

**Comment by brendankenny** [comment]
ok, no clue what's going on with github. Merged in 161519a60dea58afc5f42f450925b966da05ffcb, PR isn't closing, so closing it manually :)

**Pull request closed without merging**